suite-native/qr-code/src/components/XpubQRCodeBottomSheet.tsx

```diff
@@ -18,6 +18,7 @@ const networkTypeToSheetTitleMap: Record<NetworkType, string> = {
     cardano: 'Public key (XPUB)',
     ethereum: 'Receive address',
     ripple: 'Receive address',
+    solana: 'Public key (XPUB)',
```

Comment:
Not sure what this does but it IMO should say `Receive address` or just `Public key` for Solana (they're the same thing just with different encoding).